e2e/_suites/kubernetes-autodiscover/autodiscover_test.go

```diff
@@ -472,6 +464,9 @@ func InitializeTestSuite(ctx *godog.TestSuiteContext) {
 		// init logger
 		config.Init()
 
+		defaultEventsWaitTimeout = defaultEventsWaitTimeout * time.Duration(common.TimeoutFactor)
```

Comment:
Because Golang does not warrantee the initialisation order of the init functions, we must move this to the setupSuite, which we are sure will happen after that.

Reply:
There are some guarantees in order of init executions. In this case I think that `common.TimeoutFactor` should have been initialized before `InitializeTestSuite` is called. Wasn't it?

Reply:
Yes, in this case we moved the defaultEventsXXX timeouts to the InitializeTestSuite for that reason, so it's guaranteed that the default value is read from the environment.